osis-core/src/main/java/com/scality/osis/annotation/NotImplement.java

```diff
@@ -1,9 +1,10 @@
 /**
  *Copyright 2020 VMware, Inc.
```

Comment:
can we remove this line if we don't use vmware packages

Reply:
Its still VMware code, so we should keep it.
Most of it is copied over and modified, so we need to keep both licencnes.